integration/docker/run_test.go

```diff
@@ -204,3 +204,34 @@ var _ = Describe("run", func() {
 		})
 	})
 })
+
+var _ = Describe("run", func() {
+	var (
+		args   []string
+		id     string
+		period uint
+		quota  uint
+		vCPUs  uint
+	)
+
+	BeforeEach(func() {
+		quota = 50000
+		period = 20000
+		vCPUs = (quota + period - 1) / period
+		id = RandID(30)
+		args = []string{"--rm", "--name", id, "--cpu-quota", fmt.Sprintf("%d", quota),
+			"--cpu-period", fmt.Sprintf("%d", period), Image, "nproc"}
```

Comment:
This looks fine, but I think it would be good to test the vcpu count for various quota and period values including values which would nominally result in a vcpu count of `< 0`, `0`, `1`, `$cpus` and `$cpus + 1`, where `$cpus == $number_of_host_cpus`. That will test boundary conditions, ensure that a reasonable default gets chosen (or a reasonable error gets returned for unreasonable quota/period values), and also will ensure that we didn't just "get lucky" where `nproc` in the container happens to match `nproc` on the host.